Compiling using the production mode profile fails

If I compile a vaadin project using the provided `production-mode` profile (use the vaadin starter) then I get the following error:

`[ERROR] error: Error: ENOENT: no such file or directory, open 'C:\Users\xxx\projectpath\target\webjars\progressbar.js\1.0.1\dist\progressbar.js'`

--- a/pom.xml
+++ b/pom.xml
@@ -72,7 +72,7 @@
              it does not resolve, go to https://www.webjars.org and deploy it as "Bower GitHub"
              -->
         <dependency>
-            <groupId>org.webjars.bower</groupId>
+            <groupId>org.webjars.bowergithub.kimmobrunfeldt</groupId>
             <artifactId>progressbar.js</artifactId>
             <version>1.0.1</version>
         </dependency>

--- a/src/main/resources/META-INF/resources/frontend/com/github/appreciated/progressbar/progressbar-circle.html
+++ b/src/main/resources/META-INF/resources/frontend/com/github/appreciated/progressbar/progressbar-circle.html
@@ -1,5 +1,5 @@
 <link rel="import" href="../../../../bower_components/polymer/polymer-element.html">
-<script src="../../../../../webjars/progressbar.js/1.0.1/dist/progressbar.js"></script>
+<script src="../../../../bower_components/progressbar.js/dist/progressbar.js"></script>
 <dom-module id="progressbar-circle">
     <template>
         <style>

